doc/faq.md

```diff
@@ -66,16 +67,15 @@ To address the issue director crash or getting upgraded, some implementation int
 2. connection-table is per-lcore, when sync-ed to another DPVS, how to handle per-lcore table in another machine?
 3. "Session-Sharing" works well for "long-connection" but not "short-connection", since the connection's creation/destruction are too frequently to be sync-ed to other directors.
 
-As I know, some L4 LB implemented "session sharing" or "session synchronization", they are configuring same LIPs for each LB director. And each LIP is confgured for one CPU core. Both cases are quite different for DPVS implementation and deployment.
+As I know, some L4 LB implemented "session sharing" or "session synchronization", they are configuring same LIPs for each LB director. And each LIP is configured for one CPU core. Both cases are quite different from DPVS implementation and deployment.
```

Comment:
**. They configure the** same ...
Both cases are quite different from **DPVS's** implementation